workflows/podtato-head/workflow.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: argowf-podtato-head-chaos-
+  namespace: litmus
+spec:
+  entrypoint: argowf-chaos
+  serviceAccountName: argo-chaos
+  securityContext:
+    runAsUser: 1000
+    runAsNonRoot: true
+  arguments:
+    parameters:
+      - name: adminModeNamespace
+        value: "litmus"
+  templates:
+    - name: argowf-chaos
+      steps:
+        - - name: install-application
+            template: install-application
+        - - name: install-chaos-experiments
+            template: install-chaos-experiments
+        - - name: pod-delete
+            template: pod-delete
+        - - name: revert-chaos
+            template: revert-chaos
+          #- name: delete-application
+          #  template: delete-application
+
+    - name: install-application
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace={{workflow.parameters.adminModeNamespace}}","-typeName=resilient","-operation=apply","-timeout=400", "-label=helloservice"]
+
+    - name: install-chaos-experiments
+      container:
+        image: alpine/k8s:1.18.2
+        command: [sh, -c]
+        args:
+          - "kubectl apply -f https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/experiments.yaml -n
+            {{workflow.parameters.adminModeNamespace}} ; sleep 30"
+
+    - name: pod-delete
+      inputs:
+        artifacts:
+          - name: pod-delete
+            path: /tmp/chaosengine.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                kind: ChaosEngine
+                metadata:
+                  name: podtato-head-pod-delete-chaos
+                  namespace: {{workflow.parameters.adminModeNamespace}}
+                  labels:
+                    litmuschaos.io/engine-context: lab
+                spec:
+                  appinfo:
+                    appns: {{workflow.parameters.adminModeNamespace}}
+                    applabel: 'name=podtato-head'
```

Comment:
here we are using `name=podtato-head` label and in test-tool we are using `app=helloservice`. Can we use same labels at both places?

Reply:
Yes, 
Here we are using name. For the same we had this to sock-shop also. Since is having unique label is name and app will be conman for all pods. In pod-tato we have only one service so..